azure-iot-device/samples/README.md

```diff
@@ -99,17 +99,18 @@ This directory contains samples showing how to use the various features of the M
 
 ## Additional Samples
 
-Further samples with more complex IoT Hub scenarios are contained in the [advanced-hub-scenarios](advanced-hub-scenarios) directory, including:
+Further samples with more complex IoT Hub scenarios are contained in the [advanced-hub-scenarios](async-hub-scenarios) directory, including:
 
 * Send multiple telemetry messages from a Device
 * Receive Cloud-to-Device (C2D) messages on a Device
 * Send and receive updates to device twin
 * Receive direct method invocations
 
-Further samples with more complex IoT Edge scenarios are contained in the [advanced-edge-scenarios](advanced-edge-scenarios) directory, including:
+Further samples with more complex IoT Edge scenarios involving IoT Edge modules and downstream devices are contained in the [advanced-edge-scenarios](async-edge-scenarios) directory, including:
```

Comment:
nitpick: call it what it is "[async-edge-scenarios](async-edge-scenarios)" directory instead of advanced-edge-scenarios. Taking this opportunity to update it